cmd/authctl/user/disable.go

```diff
@@ -0,0 +1,35 @@
+package user
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/spf13/cobra"
+	"github.com/ubuntu/authd/internal/consts"
+	"github.com/ubuntu/authd/internal/proto/authd"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/credentials/insecure"
+)
+
+var DisableCmd = &cobra.Command{
+	Use:   "disable",
+	Short: "Disable a user to log in",
```

Comment:
"Disable a user to log in" is not a correct sentence. Also, I would like to convey that this can only disable users managed by authd (which is also an opportunity to at least implicitly tell users that the user accounts created by authd are managed separately from other user accounts on the system), so how about:

```suggestion
	Short: "Disable a user managed by authd",
```